src/overlays/actors/ovl_En_Dy_Extra/z_en_dy_extra.c

```diff
@@ -21,15 +30,103 @@ const ActorInit En_Dy_Extra_InitVars = {
     (ActorFunc)EnDyExtra_Update,
     (ActorFunc)EnDyExtra_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Dy_Extra/EnDyExtra_Destroy.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Dy_Extra/EnDyExtra_Init.s")
+extern UNK_TYPE D_0601BFB0;
+extern Gfx* D_0601C160;
+
+void EnDyExtra_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+}
+
+void EnDyExtra_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnDyExtra* this = THIS;
+
+    osSyncPrintf("\n\n");
+    //"Big fairy effect"
```

Comment:
Make sure to run the formatter on the file
```suggestion
    // "Big fairy effect"
```